libraries/Microsoft.Bot.Builder.Ai.Translation/Model/DetectedLanguageModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Ai.Translation.Model
+{
+    public class DetectedLanguageModel
+    {
+        [JsonProperty("language")]

```

Comment:
I suggest adding some documentation for the new classes to become easier to understand.